src/vuln_analysis/utils/functions_parsers/python_functions_parser.py

```diff
@@ -403,7 +389,24 @@ def is_script_language():
     @staticmethod
     def get_constructor_method_name():
         return '__init__'
-    
+
+    def is_package_imported(self, code_content: str, identifier: str, callee_package: str = "") -> bool:
+        for line in code_content.split(os.linesep):
+            if not self.is_comment_line(line):
+                if 'import' in line and callee_package in line:
+                    if identifier:
+                        if identifier == callee_package:
+                            if all(word in line for word in ['import', callee_package]):
+                                return True
+                        else:
+                            if all(word in line for word in ['import', callee_package, 'as', identifier]) \
+                                    or all(word in line for word in ['from', callee_package, 'import', identifier]):
+                                return True
+                    else:
+                        if all(word in line for word in ['from', callee_package, 'import']):
+                            return True
+        return False
+
```

Comment:
@tmihalac This one and refactoring of callers were a good decision, it was simplified the readability of the code